src/courses/templates/courses/mark_calculations.html

```diff
@@ -118,6 +118,15 @@ <h3>Explanation</h3>
         </li>
       </ul>
 
+      <h3>Current Mark Ranges by XP</h3>
```

Comment:
This should be below the charts:
![image](https://github.com/bytedeck/bytedeck/assets/10604391/48707cda-e927-453a-b393-933a3a22ecda)

Reply:
done